generative-ai/prompts/AI-Tutor.md

```diff
@@ -0,0 +1,261 @@
+# Design of AI Tutor Prompt
```

Comment:
Could you add a section in the REAdeme pointing to this file? Or maybe we could just merge this into the readme as a new section?

Reply:
I prefer the previous one, as we ensure the AI tutor prompt is easy to extend. Thanks. I will submit a new PR.